src/main/res/values/themes.xml

```diff
@@ -38,7 +38,6 @@
             <item name="colorControlHighlight">@color/simple_alpha33</item> 
         <item name="android:textColorPrimary">@color/white</item>
         <item name="alertDialogTheme">@style/AppCompatAlertDialogStyleDark</item>
-        <item name="actionOverflowButtonStyle">@style/WhiteActionButton</item>
```

Comment:
This was a remnant from before this approach, when I was fixing the toolbars element by element.